[TF] optimization continued

abhinavsp0730 opened the pull request:
*Description of changes:*


By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.
Notebook covered
1. convexity
2. gd
3. sgd

mli commented:
Job d2l-en/PR-1262/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1262/

terrytangyuan commented on chapter_optimization/sgd.md:
````diff
@@ -44,7 +53,7 @@ This means that, on average, the stochastic gradient is a good estimate of the g
 
 ```{.python .input}
-#@tab all
+# removed tab all because code isn't same for tf
````
No need to add comment here

terrytangyuan commented on chapter_optimization/sgd.md:
````diff
@@ -62,6 +71,45 @@ lr = (lambda: 1)  # Constant learning rate
 
+```{.python .input}
+#@tab pytorch
+# removed tab all because code isn't same for tf
````
No need to comment here in PyTorch tab

abhinavsp0730 commented:
Actually I've added because previously it's under #@tab all. Should I
removed the comments or It'll removed when doing the polishing?

On Mon 27 Jul, 2020, 3:43 PM Yuan Tang, <notifications@github.com> wrote:

> *@terrytangyuan* requested changes on this pull request.
> ------------------------------
>
> In chapter_optimization/sgd.md
> <https://github.com/d2l-ai/d2l-en/pull/1262#discussion_r460787592>:
>
> > @@ -44,7 +53,7 @@ This means that, on average, the stochastic gradient is a good estimate of the g
>  Now, we will compare it to gradient descent by adding random noise with a mean of 0 and a variance of 1 to the gradient to simulate a SGD.
>
>  ```{.python .input}
> -#@tab all
> +# removed tab all because code isn't same for tf
>
> No need to add comment here
> ------------------------------
>
> In chapter_optimization/sgd.md
> <https://github.com/d2l-ai/d2l-en/pull/1262#discussion_r460787909>:
>
> > @@ -62,6 +71,45 @@ lr = (lambda: 1)  # Constant learning rate
>  d2l.show_trace_2d(f, d2l.train_2d(sgd, steps=50))
>  ```
>
> +```{.python .input}
> +#@tab pytorch
> +# removed tab all because code isn't same for tf
>
> No need to comment here in PyTorch tab
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/d2l-ai/d2l-en/pull/1262#pullrequestreview-455651785>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKM6BKZFMWSFWV5ERLCO7T3R5VHLTANCNFSM4PIL3JKA>
> .
>

terrytangyuan commented:
I'll leave that to @mli @astonzhang to comment.

mli commented:
Job d2l-en/PR-1262/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-en/PR-1262/

abhinavsp0730 commented:
I guess the comment is unnecessary. Yuan please review now.

On Mon 27 Jul, 2020, 5:07 PM Mu Li, <notifications@github.com> wrote:

> Job d2l-en/PR-1262/3 is complete.
> Check the results at http://preview.d2l.ai/d2l-en/PR-1262/
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/d2l-ai/d2l-en/pull/1262#issuecomment-664344634>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AKM6BKZMM6NT5M4H3YGYYXDR5VRGZANCNFSM4PIL3JKA>
> .
>
